File: KampIntro/KampIntro_Odevler/Gun_05_Odev_05/Gun_05_Odev_05/Abstract/BasePlayerManager.cs
using Gun_05_Odev_05.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Gun_05_Odev_05.Abstract
{
    public abstract class BasePlayerManager : IPlayerService
    {
        public virtual void Add(Player player)
        {
            Console.WriteLine("Oyuncu veri tabanına eklendi : " + player.PlayerFirstName + " " + player.PlayerLastName);
        }
        public virtual void Update(Player player)
        {
            Console.WriteLine("Oyuncu bilgileri güncellendi : " + player.PlayerFirstName + " " + player.PlayerLastName);
        }
        public virtual void Delete(Player player)
        {
            Console.WriteLine("Oyuncu silindi : " + player.PlayerFirstName + " " + player.PlayerLastName);
        }

    }
}

File: Java_React_Ödevler/Java_React_Ödev_Gün_04_Ödev_03_OyunSatis/Gun_04_Odev_03/Abstract/BasePlayerManager.java

public abstract class BasePlayerManager implements PlayerService {

	public void add(Player player)
    {
        System.out.println("Oyuncu veri tabanına eklendi : " + player.getFirstName() + " " + player.getLastName());
    }

    public void delete(Player player)
    {
    	System.out.println("Oyuncu veri tabanından silindi : " + player.getFirstName() + " " + player.getLastName());
    }

    public void update(Player player)
    {
    	System.out.println("Oyuncu bilgileri güncellendi : " + player.getFirstName() + " " + player.getLastName());
    }
}